src/Sulu/Bundle/MediaBundle/Tests/Application/S3Kernel.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Tests\Application;
+
+use Sulu\Bundle\MediaBundle\Tests\Functional\Mock\S3AdapterMock;
+use Sulu\Component\Symfony\CompilerPass\OverwriteClassCompilerPass;
+use Sulu\Component\Symfony\CompilerPass\SetPublicCompilerPass;
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class S3Kernel extends Kernel
+{
+    public function registerContainerConfiguration(LoaderInterface $loader)
+    {
+        parent::registerContainerConfiguration($loader);
+
+        $loader->load(__DIR__ . '/config/config_s3.yml');
+    }
+
+    protected function build(ContainerBuilder $container)
+    {
+        $container->addCompilerPass(new SetPublicCompilerPass('sulu_media.storage.s3'));
+        $container->addCompilerPass(new SetPublicCompilerPass('sulu_media.storage.s3.adapter'));
+        $container->addCompilerPass(
+            new OverwriteClassCompilerPass('sulu_media.storage.s3.adapter', S3AdapterMock::class)
+        );
```

Comment:
Why don't we add these simple lines right in here? Then we wouldn't have to write two more compiler passes with tests and so on. I think that would be simpler to understand, and I don't think that we want to reuse these compiler passes anyway...